<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>28</y>
      <width>331</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Select Mission&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_mission">
       <property name="text">
        <string>New Mission</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_mission">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Loaded mission directory should contain an InOut directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Load Mission</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_path">
         <property name="text">
          <string>PATH:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="mission_path">
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>201</y>
      <width>331</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Mission Configuration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Warning">
         <property name="text">
          <string>***Apply changes in Simulation before continuing!***</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ORB_Menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Orbit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SPC_Menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Spacecraft</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SIM_Menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Simulation</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="generic_settings">
         <item row="0" column="0">
          <widget class="QPushButton" name="GRH_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Graphics</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="FOV_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FOV</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="TDR_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>TDRS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="RGN_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Region</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="IPC_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>IPC</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="NOS_Menu">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>NOS3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="DSM_Menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DSM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NDS_Menu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Nodes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
